tensorboard/defs/defs.bzl

```diff
@@ -54,19 +60,23 @@ def tf_js_binary(name, compile, deps, visibility = None, **kwargs):
         **kwargs
     )
 
-    terser_minified(
-        name = internal_min_name,
-        src = internal_rollup_name,
-        config_file = "//tensorboard/defs:terser_config.json",
-        visibility = ["//visibility:private"],
-        sourcemap = False,
-    )
+    if not dev_mode_only:
```

Comment:
nit: Inverting the control flow here (`if dev_mode_only:`) both makes
the shorter block come first and avoids a negation, both of which are
generally considered easier to read.

Other than that, this logic looks good to me.